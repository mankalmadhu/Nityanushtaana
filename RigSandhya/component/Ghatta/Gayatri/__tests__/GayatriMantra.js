import React from 'react';
import ಗಾಯತ್ರೀಮ೦ತ್ರ from '../GayatriMantra';
import Renderer from 'react-test-renderer';


jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');

test("ಗಾಯತ್ರೀಮ೦ತ್ರ", () => {
   expect(Renderer.create(
    <ಗಾಯತ್ರೀಮ೦ತ್ರ />
  ).toJSON()).toMatchSnapshot();

});