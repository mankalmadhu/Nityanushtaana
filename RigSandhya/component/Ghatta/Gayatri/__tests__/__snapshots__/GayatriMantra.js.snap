// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಗಾಯತ್ರೀಮ೦ತ್ರ 1`] = `
<ವಿಷಯ
  ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
>
  ಓ೦ ತತ್ಸ॑ವಿ॒ತುರ್ವರೇ᳚ಣ್ಯ॒೦ ಭರ್ಗೋ᳚ ದೇ॒ವಸ್ಯ॑ ಧೀಮಹಿ । ಧಿಯೋ॒ ಯೋ ನಃ॑ ಪ್ರಚೋ॒ದಯಾ᳚ತ್ ॥
</ವಿಷಯ>
`;
